daemon/cluster/executor/container/controller.go

```diff
@@ -462,53 +459,33 @@ func (r *controller) Logs(ctx context.Context, publisher exec.LogPublisher, opti
 		}
 	)
 
-	brd := bufio.NewReader(rc)
 	for {
-		// so, message header is 8 bytes, treat as uint64, pull stream off MSB
-		var header uint64
-		if err := binary.Read(brd, binary.BigEndian, &header); err != nil {
-			if err == io.EOF {
-				return nil
-			}
-
-			return errors.Wrap(err, "failed reading log header")
+		msg, ok := <-msgs
```

Comment:
This code seems to ignore `msg.Err`. I'm not sure there's anything useful that can be done with that error string, but I thought I'd ask anyway.